src/fake-timers-src.js

```diff
@@ -886,6 +894,14 @@ function withGlobal(_global) {
                     }
                 }
             }
+            if (clock.timersModuleMocks) {
```

Comment:
They are not [actual mocks](https://www.martinfowler.com/articles/mocksArentStubs.html), though, so could trim that bit off.

To be consistent with the `methods` prop, just name it `timersModuleMethods`